source/extensions/filters/listener/proxy_protocol/proxy_protocol.h

```diff
@@ -38,9 +40,11 @@ class Config {
 
 typedef std::shared_ptr<Config> ConfigSharedPtr;
 
+enum ProxyProtocolVersion { kUnknown = -1, kInProgress = -2, kV1 = 1, kV2 = 2 };
```

Comment:
nit: remove "k" prefix from constants.

Reply:
ok will work on below changes. the 'k' is from:
https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
Sorry yeah that's not something we do in Envoy, probably should be in the style guide if it's not already.